src/ComponentHelper.js

```diff
@@ -69,7 +69,24 @@ module.exports.findDirtyComponents = function(el, syncedComps, cachedData) {
     }
 
     var oldCompData = cachedData[compKey];
-    if (!deepEqual(oldCompData, newCompData)) {
+    if (compKey.endsWith("---position---") || compKey === "position"
+        || compKey.endsWith("---scale---") || compKey === "scale"){
+      const epsilon = 0.02;
```

Comment:
Hmm these probably needs to be configurable in the schema. I would think by default it shold just be doing normal equality check and then specific components could opt into a more specialized one.